saleor/checkout/complete_checkout.py

```diff
@@ -662,14 +662,16 @@ def _get_order_data(
     site_settings: "SiteSettings",
 ) -> dict:
     """Prepare data that will be converted to order and its lines."""
+    tax_configuration = checkout_info.channel.tax_configuration
```

Comment:
Maybe we should include `tax_configuration` in the `CheckoutInfo` as a separate field, as it's fetched multiple times from the `checkout_info.channel`?